src/renderer/components/editor.vue

```diff
@@ -156,12 +156,18 @@
       this.$nextTick(() => {
         const ele = this.$refs.editor
         const {
-          theme, focus: focusMode, markdown, preferLooseListItem, typewriter,
-          autoPairBracket, autoPairMarkdownSyntax, autoPairQuote
+          theme, focus: focusMode, markdown, preferLooseListItem, typewriter, autoPairBracket, autoPairMarkdownSyntax, autoPairQuote, bulletListMarker
```

Comment:
I think you should split the line - like below.

Reply:
ok